src/main/java/leo/task/TaskList.java

```diff
@@ -0,0 +1,93 @@
+package leo.task;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+public class TaskList implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    private ArrayList<Task> tasks = new ArrayList<>();
+
+    private void addTask(String taskString) throws LeoTaskException {
+        try {
+            Task task = Task.createTask(taskString);
+
+            if (task == null) {
+                throw new LeoTaskException("I'm sorry, I don't know what you want. ¿Que miras bobo?\n");
+            }
+
+            tasks.add(task);
+            System.out.printf("Alright, I've added: %s to your task list.\n", tasks.get(tasks.size() - 1));
+            System.out.printf("You have %d tasks in your list, vamos, get moving!\n", tasks.size());
+        } catch (LeoTaskException e) {
+            e.printStackTrace();
+            System.out.println();
+        }
+    }
+
+    private void deleteTask(int taskId) throws LeoTaskException {
```

Comment:
I like how you lower cased the abbreviation in the variable name taskId, as you avoided the common mistake of naming it as TaskID instead!